src/main/java/com/puppycrawl/tools/checkstyle/api/JavadocCommentsTokenTypes.java

```diff
@@ -864,7 +864,20 @@ public final class JavadocCommentsTokenTypes {
     public static final int EXTENDS = JavadocCommentsLexer.EXTENDS;
 
     /**
-     * Keyword {@code super} in type parameters.
+     * {@code SUPER} Keyword {@code super} in type parameters.
+     *
+     * <p>Represents the {@code super} keyword in generics.</p>
+     *
+     * <p><b>Example:</b></p>
+     * <pre>{@code * <? super T>}</pre>
```

Comment:
please fin in web real use case of "super" in javadoc, make it compact and use as example.